### DISCUSSION TITLE
Can I search in PDFs?

### DISCUSSION THREAD

**Discussion by Gnopps** [TriliumNext/Trilium#1378] [Q&A] [answered]
Trilium does look great and I noticed that I can search in a PDF-document by opening it in Trilium and selecting "Search in note". However, if I use the global search then results from within the PDF are note returned. Is this possible to enable? Basically that global search would include results from within attached PDF-docs.

**Comment by zadam** [comment, marked as answer]
Hi, this is not supported and it's not a question of "enabling" it - this would have to be implemented as a new feature with something like pdf.js extracting all text nodes which would then be searched over.

**Reply by Gnopps** [reply]
Great answer, thank you!
